### PR TITLE
Fix nanoGPT sample reshape

### PR DESCRIPTION
**Pull request by yizhuoz004**

### CODE REVIEW

File: tripy/examples/nanogpt/model.py [hunk 1]
```diff
@@ -76,19 +75,15 @@ def __init__(self, config):
         self.zeros = tp.zeros((1, 1, self.seq_len, self.seq_len), dtype=config.dtype)
 
     def __call__(self, x: tp.Tensor):
-        B, T, C = tp.reshape(x.shape[0], (1,)), tp.reshape(x.shape[1], (1,)), tp.reshape(x.shape[2], (1,))
+        B, T = x.shape[0], x.shape[1]
```
Comment (pranavm-nvidia) [inline tripy/examples/nanogpt/model.py:78]:
```suggestion
        B, T = x.shape[0:2]
```

Comment (pranavm-nvidia) [reply tripy/examples/nanogpt/model.py:78]: Why do we call it `T` here? Isn't it sequence length? Usually that's `S`.

File: tripy/examples/nanogpt/model.py [hunk 2]
```diff
@@ -108,9 +103,7 @@ def __call__(self, x: tp.Tensor):
         # (batch_size, num_heads, seq_len, seq_len) @ (batch_size, num_heads, seq_len, head_size) -> (batch_size, num_heads, seq_len, head_size)
         out = att @ v
         out = tp.cast(out, x.dtype)
-        out_shape = tp.concatenate([B, T, tp.Tensor([self.embedding_size])], dim=0)
-        # WAR to prevent computing output rank in infer_rank for reshape, will be addressed by #228
-        out_shape.trace_tensor.shape = (3,)
+        out_shape = [B, T, self.embedding_size]
         out = tp.reshape(tp.transpose(out, 1, 2), out_shape)
```
Comment (pranavm-nvidia) [inline tripy/examples/nanogpt/model.py:107]: We can probably just inline the shapes here and above

**Review by pranavm-nvidia — APPROVED** [review]

File: tripy/examples/nanogpt/model.py (hunk 1 above)
Comment (yizhuoz004) [reply tripy/examples/nanogpt/model.py]: Yes it is sequence length, I think it's pretty common to use `T`, nanoGPT uses T: https://github.com/karpathy/nanoGPT/blob/master/model.py#L53

File: tripy/examples/nanogpt/model.py (hunk 2 above)
Comment (yizhuoz004) [reply tripy/examples/nanogpt/model.py]: Updated.

**Pull request merged**